ta/pkcs11/src/processing_ec.c

```diff
@@ -0,0 +1,499 @@
+// SPDX-License-Identifier: BSD-2-Clause
+/*
+ * Copyright (c) 2017-2020, Linaro Limited
+ */
+
+#include <assert.h>
+#include <pkcs11_ta.h>
+#include <tee_api_defines.h>
+#include <tee_internal_api.h>
+#include <util.h>
+
+#include "attributes.h"
+#include "processing.h"
+
+/*
+ * DER encoded EC parameters generated with script:
+ *   ta/pkcs11/scripts/dump_ec_params.sh
```

Comment:
correct path is `ta/pkcs11/scripts/dump_ec_curve_params.sh `

Reply:
Fixing.